microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -534,6 +534,36 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image<!-- todo: is the menuitem capital I or lowercase?  global in section -->
+
+To use the Magnify Image feature, right-click an image and then select **Magnify Image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
+
+
+<!-- ------------------------------ -->
+#### AI-based image enhancement
+
+If your browser has WebNN enabled, there's also a button<!-- todo: what's the button label?  can we commit to specifying the button label at this time?  [click] the **Foo** button --> to use AI to enhance the image.  To check whether WebNN is enabled, go to the `edge://gpu` page, and then in the **Graphics Feature Status** section, see whether **WebNN** says **Enabled**:<!-- todo: how to control whether WebNN is enabled? -->
+
+![The status of the WebNN setting, shown in the edge://gpu page](./index-images/webnn-disabled-gpu.png)
```

Comment:
keep or delete screenshot?
@apryzgoda @sjdallst2 
rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

Reply:
i find the screenshot helpful & interesting, keeping.  if there were too many, could remove it in that case.

Reply:
We don't usually put features in the whitepaper until they're released to stable and enabled by default. I think that the screenshot should be removed, and we should wait to post this until it's enabled by default.

Reply:
png removed per other comment.